prog-o-meter.py

```diff
@@ -352,7 +392,175 @@ def save_and_close(self, event=None):
+        Create 3 Buttons:
+            defaults_button: 
+                when pressed uses the default_values function
+            apply_button:    
+                when pressed uses the apply_values function
+            back_button:     
+                when pressed uses the go_back function
+        Obs:
+            The Notebook widget cant be used without ttk,
+            Every other thing has it Tk counterpart
+        """
+        root_tab = ttk.Notebook(self.root)
+        self.colors_tab = ttk.Frame(root_tab)   
+        self.motivation_tab = ttk.Frame(root_tab)   
+        self.days_tab = ttk.Frame(root_tab)
+        root_tab.add(self.colors_tab, text="Colors")
+        root_tab.add(self.motivation_tab, text="Motivations")
+        root_tab.add(self.days_tab, text="Days")
+        root_tab.grid(row=0, column=0, columnspan=3)
+        defaults_button = ttk.Button(self.root, text="Default",
+                                     command=self.default_values)
+        defaults_button.grid(row=1, column=0, sticky="WE")
+        apply_button = ttk.Button(self.root,
+                                  text="Apply",command=self.apply_values)
+        apply_button.grid(row=1, column=1, sticky="WE")
+        back_button = ttk.Button(self.root, text="Back", command=self.go_back)
+        back_button.grid(row=1, column=2, sticky="WE")
+
+    def colors_layout(self):
+        """The color tab, It cointains Labels and Radio Buttons
+
+        uses lists for the colors and a dict to get the default positions
+        It uses grid() instead of pack() for better positioning
+        """
+        self.primary_label = ttk.Label(self.colors_tab, text="Primary Color")
+        self.primary_label.grid(row=0, column = 4, sticky = "WE")
+        colors = ["Red", "Orange", "Yellow", "Green", "Light Blue", "Blue",
+                  "Purple", "Black", "Rainbow"]
+        colorslist = {"#FF0000": 0, "#FF4F00": 1,
+                           "#FFFF00": 2, "#008000": 3,
+                           "#BFEFFF": 4, "#0000FF": 5,
+                           "#800080": 6, "#000000": 7,
+                           "#RAINBOW": 8}
+        self.primary = Tk.IntVar()
+        self.secondary = Tk.IntVar()
+        self.primary.set(colorslist[self.user.get_color_primary()])
+        self.secondary.set(colorslist[self.user.get_color_secondary()])
+        c=0        #amount of columns
+        r=1        #row number
+        v=0        #the number value
+        for color in colors:    #a for loop that creates the buttons
+            ttk.Radiobutton(self.colors_tab, text=color,
+                        variable=self.primary, value=v).grid(row=r, column=c)
+            v = v + 1
+            c = c + 1
+        c = 0
+        r = 3
+        v = 0
+        self.secondary_label = ttk.Label(self.colors_tab, text="Secondary Color")
+        self.secondary_label.grid(row=2, column=4)
+        for color in colors:        #another for loop instead of copy&paste

```

Comment:
this comment is good at explaining your choice, but since the code will constantly keep changing when people are adding more and more stuff, as time passes, it may become a little unclear what this references to. I think it might be better to just explain what the for-loop does, is that ok with you? :)

Reply:
Alright.